pkg/cmd/roachtest/activerecord.go

```diff
@@ -20,7 +20,8 @@ import (
 
 var activerecordResultRegex = regexp.MustCompile(`^(?P<test>[^\s]+#[^\s]+) = (?P<timing>\d+\.\d+ s) = (?P<result>.)$`)
 var railsReleaseTagRegex = regexp.MustCompile(`^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)\.?(?P<subpoint>\d*)$`)
-var supportedRailsVersion = "5.2.4.3"
+var supportedRailsVersion = "6.0.3.4"
```

Comment:
do we want to test 5.2 as well as 6.0, or just 6.0?

Reply:
the plan was to just test 6.0 for now.